extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
   ]
 }
 
+```
+## Experimental Feature: Branching
+
+We’ve implemented additional features to allow for the construction of non-linear stories. By utilizing these features, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story.  Example use cases include quizzes, choose-your-own-story experiences, and story table of contents.
+
+Note that all branching features are behind an `amp-story-branching` experiment. To turn on the experiment, enable it in your dev console:
+
+``` AMP.toggleExperiment('amp-story-branching') ```
+
+For more information see, [experimental features](https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html?format=stories).
+
+## Manipulating Navigation
+The features listed below allow you to manipulate navigation within a story. Note that when using branching features, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. That is, if a user navigates from page-1 to page-5 and then page-6, navigating backwards will follow this exact path (page-6 to page-5 and finally page-1). The other pages will not be exposed to the user.
+
+### The `goToPage` Action
+A new `goToPage` action allows for 'jumping' from one page within a story to another. The action can be used with allowed elements in `amp-story` and their respective events. Clicking on an element with this action should bring the user to another page within the same story. Using this action **requires** that you give the `amp-story` element an id, a unique identifier, as the story itself is the target.  Read more about actions and events in AMP [here](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.html?format=stories).
```

Comment:
Question: Are there other story specific actions/events? Would it be useful to have a stories specific [actions and events document](https://amp-dev-staging.appspot.com/documentation/guides-and-tutorials/learn/amp-actions-and-events.html?format=websites)?

Reply:
I think that `goToPage` is the only story specific event. Given that the feature is still behind an experiment I'm not sure if we should add it to the actions and events document just yet. + @newmuis, in case he has any thoughts on this.